alpakaConfig.cmake

```diff
@@ -329,11 +329,12 @@ IF(ALPAKA_ACC_GPU_CUDA_ENABLE)
 
                 SET(CUDA_PROPAGATE_HOST_FLAGS ON)
 
+
                 IF(ALPAKA_CUDA_VERSION VERSION_EQUAL 8.0)
                     LIST(APPEND CUDA_NVCC_FLAGS "-Wno-deprecated-gpu-targets")
                 ENDIF()
 
-                LIST(APPEND CUDA_NVCC_FLAGS "-arch=${ALPAKA_CUDA_ARCH}")
+                LIST(APPEND CUDA_NVCC_FLAGS "-arch=${ALPAKA_CUDA_ARCH} --expt-extended-lambda")
```

Comment:
Please add this flag in an extra line and only 'IF(NOT ALPAKA_CUDA_VERSION VERSION_LESS 7.5)'